packages/block-editor/src/store/reducer.js

```diff
@@ -1298,6 +1305,14 @@ export function selection( state = {}, action ) {
 			};
 		case 'MULTI_SELECT':
 			const { start, end } = action;
+
+			if (
```

Comment:
Why do we need this?

Reply:
Because it might overwrite more specific selection with an offset. Some tests were failing without this, but forgot which ones :)